plugins/generator-1.21.1/neoforge-1.21.1/procedures/entity_set_mainhand_item.java.ftl

```diff
@@ -1,7 +1,11 @@
 <#include "mcitems.ftl">
-if (${input$entity} instanceof LivingEntity _entity) {
-	ItemStack _setstack = ${mappedMCItemToItemStackCode(input$item, 1)}.copy();
-	_setstack.setCount(${opt.toInt(input$amount)});
-	_entity.setItemInHand(InteractionHand.MAIN_HAND, _setstack);
+<@head>if (${input$entity} instanceof LivingEntity _entity) {</@head>

```

Comment:
It is very unlikely the same item will be applied to the main and the off hand, so does this optimisation make sense for this case? I would narrow down the optimisation for the outer if only and keep the tail

Applies to both MC versions.

Inner { } is also sort of unclean code